core/workflow-operator/src/main/scala/edu/uci/ics/amber/operator/state/DataToStateOpExec.scala

```diff
@@ -0,0 +1,21 @@
+package edu.uci.ics.amber.operator.state
+
+import edu.uci.ics.amber.core.executor.OperatorExecutor
+import edu.uci.ics.amber.core.marker.State
+import edu.uci.ics.amber.core.tuple.{Tuple, TupleLike}
+import edu.uci.ics.amber.util.JSONUtils.objectMapper
+
+class DataToStateOpExec(descString: String) extends OperatorExecutor {
+  private val desc: DataToStateOpDesc =
+    objectMapper.readValue(descString, classOf[DataToStateOpDesc])
+  private var stateTuple: Option[Tuple] = None
+
+  override def processTuple(tuple: Tuple, port: Int): Iterator[TupleLike] = {
+    if (stateTuple.isEmpty)
+      stateTuple = Some(tuple)
+    Iterator.empty
```

Comment:
you can only convert the first tuple, and implicitly ignored (voided) all following tuples. this has not been reflected in any documentation or operator description. please clarify it.